packages/flutter/lib/material.dart

```diff
@@ -77,7 +77,7 @@ export 'src/material/ink_well.dart';
 export 'src/material/input_border.dart';
 export 'src/material/input_decorator.dart';
 export 'src/material/list_tile.dart';
-export 'src/material/material.dart';
+export 'src/material/material.dart' hide ElevationOverlay;
```

Comment:
This is weird since we don't do it anywhere else. It seems safe, since it doesn't affect the public API.

Reply:
I just moved the `ElevationOverlay` class into its own file and didn't include it in the `material.dart` package exports. That seems to be the best way to handle this for now.